toast-tk-runtime/src/main/java/io/toast/tk/runtime/emailReport.java

```diff
@@ -0,0 +1,58 @@
+package io.toast.tk.runtime;
+
+import java.util.Properties;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.AddressException;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+/**
+ * @author Crunchify.com
+ * 
+ */
+
+public class emailReport {
+
+	static Properties mailServerProperties;
+	static Session getMailSession;
+	static MimeMessage generateMailMessage;
+
+	public static void main(String args[]) throws AddressException, MessagingException {
+		generateAndSendEmail();
+		System.out.println("\n\n ===> Your Java Program has just sent an Email successfully. Check your email..");
+	}
+
+	public static void generateAndSendEmail() throws AddressException, MessagingException {
+
+		// Step1
+		System.out.println("\n 1st ===> setup Mail Server Properties..");
+		mailServerProperties = System.getProperties();
+		mailServerProperties.put("mail.smtp.port", "993");
+		mailServerProperties.put("mail.smtp.auth", "true");
+		mailServerProperties.put("mail.smtp.starttls.enable", "true");
+		System.out.println("Mail Server Properties have been setup successfully..");
+
+		// Step2
+		System.out.println("\n\n 2nd ===> get Mail Session..");
+		getMailSession = Session.getDefaultInstance(mailServerProperties, null);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Synchronize this lazy initialization of 'getMailSession' [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AS2444)